### PR TITLE
add error return for new connection init

### PR DESCRIPTION
**Pull request by halilkocaoz**
#22

### CODE REVIEW

File: connector.go [hunk 1]
```diff
@@ -83,7 +83,11 @@ func newConnector(configPath string, mapper Mapper, logger logger.Logger, errorL
 	if err != nil {
 		connector.errorLogger.Printf("Dcp error: %v", err)
 	}
+
 	connector.dcp = dcp
-	connector.producer = kafka.NewProducer(c.Kafka, connector.logger, connector.errorLogger, dcp.Commit)
-	return connector
+	connector.producer, err = kafka.NewProducer(c.Kafka, connector.logger, connector.errorLogger, dcp.Commit)
+	if err != nil {
+		panic("Kafka producer is could not initialized: " + err.Error())
```
Comment (mhmtszr) [inline connector.go:90]: We may also remove this panic what do you think?

Comment (halilkocaoz) [reply connector.go:90]: yeah, we can remove the panic from the connector.go and use panic at example. also i have question, if error turns out while initializing new dcp should, should we return the method with err after the logging?

Comment (mhmtszr) [reply connector.go:90]: I think yes, we can log and after that, we can return err.

Comment (halilkocaoz) [reply connector.go]: I did some changes about new connector method returnings

**Review by mhmtszr — APPROVED** [review]
Health your hands.

**Pull request merged**